modules/nf-core/merqury/merqury/main.nf

```diff
@@ -26,6 +26,7 @@ process MERQURY {
     tuple val(meta), path("${prefix}.qv")        , emit: assembly_qv
     tuple val(meta), path("${prefix}.*.qv")      , emit: scaffold_qv
     tuple val(meta), path("*.hist.ploidy")       , emit: read_ploidy
+    tuple val(meta), path("*.hapmers.blob.png")  , emit: hapmers_blob_png           , optional: true
```

Comment:
Can we add this into the test too please?

Reply:
It is added: https://github.com/nf-core/modules/blob/0e2e61c52586421921436e10899f58050e9d2c8b/modules/nf-core/merqury/merqury/tests/main.nf.test#L78

This file is generated in the trio mode. I have tested and incorporated that in [assemblyqc/dev](https://github.com/Plant-Food-Research-Open/assemblyqc/tree/dev) using the 
official test data.

I'll try to add a test here using the exisiting nf-core/test data.

Reply:
I have added a trio mode test. The prefixes of the output files are bit off in the trio mode. The reason is that we don't have control on prefix in the `MERYL/COUNT` module, and, that is because of its own reasons. With the addition of `MERQURY/HAPMERS`, the prefixes can become more intuitive.